addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```
@@ -15,17 +15,40 @@
     overflow: auto
 }
 
-.o_ComposerTextInput_mentionMenuItem {
-    white-space: nowrap;
-    overflow: hidden;
+.o_ComposerTextInput_mention_dropdown .o_mention_proposition {
 
-    &.o-mobile {
-        padding: 5px;
+    display: flex;
+    width: 100%;
+    padding: 3px 15px;
+    .o_mention_updatable_im_status {
+        padding-right: 3px;
```

```suggestion
        padding-inline-end: map-get($spacers, 2);
```
2 or 1 depending on what looks beter

Copy/paste of my about margin-left/right, but same works for padding:

For **margin-left** or **margin-right**, when it depends on the direction of the text, this would be correct assuming `ltr` but will fail when using `rtl`.

So when the margin indeed depends on the direction of the text (which is 99% of the time), we need to use `margin-inline-start` or `margin-inline-end`.

Note that maybe here is a bad example, the question to ask is do we want the content to be on the left or on the right of the text when the text direction is inversed?

---

Done